elastic/kibana
# [Discover] Fix context view for document ids containing special characters

## Summary

Fixes #122638

This PR adds decoding for encoded document id param. 

### Testing notes

1. Create a document containing a + in its id
```
PUT context/_doc/1+1=2
{
  "timestamp":"2022-01-10T09:30:23",
  "name":"test"
}
```
2. Create a data view
3. Navigate to Discover, adapt time range so you can find the document
4. Expand the document and click on "View surrounding documents"
5. Context view of anchor document should be opened without errors.
